doc/include/imagebitmask_table.rst

```diff
@@ -0,0 +1,13 @@
+==========  =========  ===================================================
```

Comment:
If it is easy to do, maybe add another column which is `2**BIT_VALUE`

Reply:
Done.